src/toil/jobStores/azureJobStore.py

```diff
@@ -713,12 +715,12 @@ def toItem(self, chunkSize=maxAzureTablePropertySize):
 def retryOnAzureTimeout(exception):
     timeoutMsg = "could not be completed within the specified time"
     busyMsg = "Service Unavailable"
-    return isinstance(exception, WindowsAzureError) and (timeoutMsg in str(exception)
-        or busyMsg in str(exception))
+    return isinstance(exception, AzureException) and (timeoutMsg in str(exception)
+                                                      or busyMsg in str(exception))
```

Comment:
wrap **before** `and` and keep the disjunction on a line. When wrapping expressions (which are trees), prefer keeping the leafs together on a line and instead wrap at higher levels in the tree.